src/main/java/net/greatstart/dto/DtoUserProfile.java

```diff
@@ -17,6 +20,7 @@
     private String email;
     private String lastName;
     private byte[] photo;
+    private List<DtoInvestment> dtoInvestments;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "dtoInvestments" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)